README.md

```diff
@@ -1,3 +1,3 @@
 
-# Nothing to see here, move along now.
+# Decentralized activity streams and more.
```

Comment:
Well, needs more text of *some kind*... oh! we definitely need a LICENSE file as well, or maybe just here say 'under whatever licesne'